include/vrt.h

```diff
@@ -58,6 +58,8 @@
  * binary/load-time compatible, increment MAJOR version
  *
  * NEXT (2024-09-15)
+ * 19.1 (xxx)
+ *	ObjWaitExtend() gained statep argument
```

Comment:
Not a VRT change, shouldn't trigger a VRT bump.

Reply:
We have a long tradition of recording many more changes in `vrt.h` than changes to functions prefixed `VRT_`, and we have not even defined what VRT really is (#3555).
Using the VRT version to mark changes like this one GREATLY simplifies use in vmods/vexts, in particular in light of the de-autotoolification at zero + epsilon cost. So what is the gain of NOT using vrt logs and versions?

Reply:
We've already been there, I find it confusing to make a VRT bump for something other than a VRT change. And that includes the systematic "better safe than sorry" systematic major (!) bump for biannual releases.

This particular function is in `cache_varnishd.h`, which should settle the VRT-or-not question in this case.

Note that I approved the pull request and didn't even try to "request changes". I'm merely mentioning that this should say "NEXT" instead of "19.1" but I'm not vetoing it.

Reply:
I agree that VEXTs are a different beast of their own.